var/spack/repos/builtin/packages/libtirpc/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Libtirpc(AutotoolsPackage):
+    """Libtirpc is a port of Suns Transport-Independent RPC library to Linux.
+    """
+
+    homepage = "https://sourceforge.net/projects/libtirpc/"
+    url      = "https://sourceforge.net/projects/libtirpc/files/libtirpc/1.1.4/libtirpc-1.1.4.tar.bz2/download"
+
+    version('1.1.4', sha256='2ca529f02292e10c158562295a1ffd95d2ce8af97820e3534fe1b0e3aec7561d')
+
+    # FIXME: build error on macOS
+    # auth_none.c:81:9: error: unknown type name 'mutex_t'
```

Comment:
This error message is really bugging me. `mutex_t` is defined in `tirpc/reentrant.h`:
```c
#define mutex_t pthread_mutex_t
```
and `reentrant.h` is included in `src/auth_none.c`:
```c
#include <reentrant.h>
```
so I don't know why it's complaining about an undefined symbol.

Reply:
Looks like it installs just fine and passes all tests on CentOS 7 with GCC 7.1.0. Not sure if macOS problem or Clang problem.